README.md

```diff
@@ -75,7 +75,7 @@ In either case, you will want to have [Serverless] installed, eg. `npm install -
 
   functions:
     myfunc:
-      handler: mypackage.myfunc
+      handler: mypackage.mypackage-exe
       # Here, mypackage is the Haskell package name and myfunc is the executable
```

Comment:
I'm not sure what you mean. That's what my change is: replacing myfunc with mypackage-exe. In the template, the executable is called myfunc, but if you use `stack new mypackage` it will create an executable called `mypackage-exe`.

Reply:
Exactly. The comment still says "myfunc is the executable" and should be updated.